airflow/providers/amazon/aws/operators/athena.py

```diff
@@ -38,8 +38,8 @@ class AWSAthenaOperator(BaseOperator):
     :type database: str
     :param output_location: s3 path to write the query results into. (templated)
     :type output_location: str
-    :param aws_conn_id: aws connection to use
-    :type aws_conn_id: <str> adding this param to template_fields so that it can be overriden using jinja template from Dags 
+    :param aws_conn_id: Aws connection to use in template_fields so that it can be overriden using jinja template from Dags. 
+    :type aws_conn_id: <str>
```

Comment:
Can you please undo your changes to the type field? You've added carets but this is not done elsewhere in the code.

Reply:
Please update all other occurrences.